src/biconnectivity/biconnect.jl

```diff
@@ -0,0 +1,70 @@
+"""
+Biconnections: A state type for Depth First Search that finds the biconnected components
+"""
+type Biconnections
+    low::Vector{Int}
+    depth::Vector{Int}
+    stack::Vector{Edge}
+    biconnected_comps::Vector{Vector{Edge}}
+    id::Int
+end
+
+function Biconnections(g::SimpleGraph)
+    n = nv(g)
+    return Biconnections(zeros(Int, n), zeros(Int, n), [], [], 0)
```

Comment:
instead of `[]`, can you use `Vector{Int}()` and `Vector{Edge}()` to be explicit here?